<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerDefineWindow</class>
 <widget class="QWidget" name="LayerDefineWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图层属性</string>
  </property>
  <property name="windowIcon">
   <iconset resource="orion2.qrc">
    <normaloff>:/images/unique32.png</normaloff>:/images/unique32.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>461</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="savepushButton">
      <property name="text">
       <string>保存</string>
      </property>
      <property name="icon">
       <iconset resource="orion2.qrc">
        <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelpushButton">
      <property name="text">
       <string>取消</string>
      </property>
      <property name="icon">
       <iconset resource="orion2.qrc">
        <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>465</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="PictureWidget" name="colorwidget" native="true">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>名称</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="namelineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>缩放变化速度</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>相对滚动速度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="clrpushButton">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>背景颜色</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="pxspinBox">
      <property name="suffix">
       <string/>
      </property>
      <property name="prefix">
       <string>水平: </string>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="pyspinBox">
      <property name="prefix">
       <string>垂直: </string>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="visiblecheckBox">
        <property name="text">
         <string>初始可见</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="transparentcheckBox">
        <property name="text">
         <string>背景完全透明</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="texturecheckBox">
        <property name="text">
         <string>缓存纹理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rendercheckBox">
        <property name="text">
         <string>是否使用渲染网格</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSlider" name="alphahorizontalSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="alphalabel">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QSpinBox" name="scalespinBox">
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureWidget</class>
   <extends>QWidget</extends>
   <header>picturewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="orion2.qrc"/>
 </resources>
 <connections/>
</ui>
